src/data_gradients/feature_extractors/object_detection/bounding_boxes_area.py

```diff
@@ -75,6 +88,56 @@ def aggregate(self) -> Feature:
         )
         return feature
 
+    @staticmethod
+    def _compute_histogram(df: pd.DataFrame, transform_name: str, min_bin_val: int = 1) -> dict:
+        """
+        Compute histograms for bounding box areas per class.
+
+        :param df: DataFrame containing bounding box data.
+        :param transform_name: Type of transformation (like 'sqrt').
+        :param min_bin_val: Minimum size value for the histogram.
+        :return: A dictionary containing relevant histogram information.
+            Example:
+            {
+                'train': {
+                    'transform': 'sqrt',
+                    'bin_width': width between histogram bins,
+                    'min_value': min size value in the histogram,
+                    'max_value': max size value in the histogram,
+                    'histograms': a dictionary of class name and its matching histogram
```

Comment:
I know I proposed this, but let's make it slightly better:
```python
{
            'train': {
                'transform': 'sqrt', # Transformation applied to the bbox area
                'bin_width': 1, # width between histogram bins,
                'min_value': 1, # min size value in the histogram,
                'max_value': 4, # max size value in the histogram,
                'histograms': { #  Dictionary of class name and its matching histogram
                    'A': [1, 0, 2],
                    'B': [0, 1, 0]
                }
            },
            'val': {
                'transform': 'sqrt',
                'bin_width': 1,
                'min_value': 1,
                'max_value': 4,
                'histograms': {
                    'A': [0, 1, 0],
                    'B': [0, 0, 1],
                    'C': [0, 0, 1]
                }
            }
        }
```

Reply:
IS this ok? [de12a03](https://github.com/Deci-AI/data-gradients/pull/180/commits/de12a03d8c8f7f7e5a5e5a2c3ca6dde7491127ae)

Reply:
Maybe:
`width between histogram bins,` -> `width between histogram bins. This is affected by transform `
`min size value in the histogram,` -> `min (transformed) area value to include in the`
`max size value in the histogram,` -> `max (transformed) area value to include in the`
'Dictionary of class name and its matching histogram.`